src/main/java/org/prebid/server/auction/mediatypeprocessor/MultiFormatMediaTypeProcessor.java

```diff
@@ -30,13 +30,24 @@ public MultiFormatMediaTypeProcessor(BidderCatalog bidderCatalog) {
 
     @Override
     public MediaTypeProcessingResult process(BidRequest bidRequest, String bidderName, Account account) {
+        final BidRequest.BidRequestBuilder bidRequestBuilder = Optional.ofNullable(bidRequest.getExt())
+                .map(ExtRequest::getPrebid)
+                .map(prebid -> prebid.toBuilder().biddercontrols(null).build())
+                .map(prebid -> {
+                    final ExtRequest extRequest = ExtRequest.of(prebid);
+                    extRequest.addProperties(bidRequest.getExt().getProperties());
+                    return extRequest;
+                })
+                .map(extRequest -> bidRequest.toBuilder().ext(extRequest))
+                .orElse(bidRequest.toBuilder());
```

Comment:
This logic is completely inappropriate here.

As a quick solution, create a new `MediaTypeProcessor` that will execute at the very end and remove unnecessary data. Also please add a TODO for this new class to make sure we refactor and get rid of it.

Reply:
idk... this service changes the bidrequest according to the multiformat cases, so don't see why it's inappropriate here.